File: Day7/Day7.js
let prompt = require('prompt-sync')();

function reverseEachWord(sentence) {
	let words = sentence.split(" ");
	let result = "";
	for(let word of words) {
		let reverse = " ";
		for(let count in word) {
			reverse = word[count] + reverse;}
			result += reverse;
	
	}
return result;
}

let sentence = prompt("Input a sentence: ");

console.log(reverseEachWord(sentence));

File: Day7/Day7.java
import java.util.Scanner;
import java.util.Arrays;
public class DaySeven {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);


System.out.print("Enter a sentence: ");
String sentence = input.nextLine();
 
System.out.print(reverseEachWord(sentence));

}

public static String reverseEachWord(String sentence) {
	String[] words = sentence.split(" ");
	String result = "";
	for(String word : words) {
		String reverse = " ";
		for(int count = 0; count < word.length(); count++) {
			reverse = word.charAt(count) + reverse;}
			result += reverse;
	}
return result;


}
}